lib/active_merchant/billing/gateways/openpay.rb

```diff
@@ -1,8 +1,15 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class OpenpayGateway < Gateway
-      self.live_url = 'https://api.openpay.mx/v1/'
-      self.test_url = 'https://sandbox-api.openpay.mx/v1/'
+      class_attribute :mx_live_url, :mx_test_url
+      class_attribute :co_live_url, :co_test_url
+
+      self.co_live_url = 'https://api.openpay.co/v1/'
```

Comment:
The code looks good but just out of curiosity why change it to `co_live_url` and `co_test_url`?

Reply:
the idea to be more verbose about the configuration of the gateway. Currently, openpay supports two domains, one for Colombia and the other for Mexico, being Colombia the default.  that's the reason.

Reply:
Okay, thanks for the explanation!